test/scripts/create_vm_libvirt.sh

```
@@ -25,6 +25,7 @@ SSH_PRIVATE_KEY_PATH="/home/${USER}/.ssh/id_rsa"
 SSH_PUBLIC_KEY_PATH="/home/${USER}/.ssh/id_rsa.pub"
 USER_DATA_FILE="user-data.yaml"
 FLIGHTCTL_PATH="/home/${USER}/flightctl"
```

Shellcheck (0.10.0)

[warning] FLIGHTCTL_PATH appears unused. Verify use (or export if used externally).

(SC2034)